var/spack/repos/builtin/packages/racon/package.py

```diff
@@ -11,9 +11,19 @@ class Racon(CMakePackage):
     """Ultrafast consensus module for raw de novo genome assembly of long
     uncorrected reads."""
 
-    homepage = "https://github.com/isovic/racon"
+    homepage = "https://github.com/lbcb-sci/racon"
     url = "https://github.com/isovic/racon/releases/download/1.2.1/racon-v1.2.1.tar.gz"
 
+    version("1.5.0", sha256="41e362f71cc03b934f17d6e2c0d626e1b2997258261b14551586de006666424a")
```

Comment:
These versions don't seem to exist on github...

Reply:
racon has been moved to a new repo. The url has been updated.